### ISSUE TITLE
Please update the rubygems.org version to the latest version.

### ISSUE THREAD

**Issue by zanloy** [kmrshntr/omniauth-slack#52]
I spent quite a while trying to figure out why my response was coming back nil before I realized the version on rubygems.org is like a year old. As soon as I pointed it to this repository, it worked great. This might help a lot more people in the future who don't want to add a bunch of github repos directory in their Gemfile.

**Comment by JustLey** [comment]
@zanloy I recent went through the issues/PRs on this repo and read somewhere that @kmrshntr will be dealing with this gems issues and release in the next weeks when he has some free time. Can't remember where, but I guess we'll have to wait. I advise you to use :github => 'kmrshntr/omniauth-slack', :ref => 'commit_number' if you need some updated code (I personally use [this commit](https://github.com/kmrshntr/omniauth-slack/commits/f050be234b0e6d5faaacb4b3c47660badaa4e1a3)

**Comment by mauriciosouto** [comment]
Hi @zanloy 

Can you share your gemfile? or any other tip.
I'm trying to use this repo (can't get user email with rubygem repo) and the only thing I can get is `Could not authenticate you from Slack because "Invalid credentials".`

Thanks

**Comment by zanloy** [comment]
@mauriciosouto I actually forked this repo because I couldn't wait for the PR to fix a common error. But here is the line in my Gemfile:

gem 'omniauth-slack', git: 'https://github.com/zanloy/omniauth-slack.git'

**Comment by mauriciosouto** [comment]
Thanks @zanloy 
I got the email working with your fork...

**Comment by adrianotadao** [comment]
I think the better is to have a new version of the gem. If we use the master, we can have problems =(

@kmrshntr could you please generate a new version? 💟

**Comment by dkniffin** [comment]
I'd also like to see the rubygems version get bumped. @kmrshntr: it looks like you're not very active on github anymore. I don't know much about omniauth, but I have managed some ruby gems in the past. I'd be happy to help maintain this one, if you're interested.

**Comment by adrianotadao** [comment]
I can help too @dkniffin ... I have a fork here but I world like to use the rubygem version

**Comment by kevinelliott** [comment]
What is going on with this?

**Comment by alfredvaa** [comment]
I got nil as uid in the response when using this gem. Turns out I had to add the github reference in my Gemfile to make it work: ```gem 'omniauth-slack', github: 'kmrshntr/omniauth-slack'```

It would be nice if a new version of the gem is released with the latest stuff.

**Comment by dkniffin** [comment]
Judging from @kmrshntr's activity, we might be looking at abandonware here...

**Comment by alfredvaa** [comment]
@dkniffin Yes, it looks like that. I really just wanted to leave the solution here if anyone else was wondering.
